### PR TITLE
[PAY-848] Fix UploadChip spacing

### PR DESCRIPTION
**Pull request by piazzatron**
### Description

- Fixes spacing above UploadChip. Tried to do this all in the NavBar component, so none of the subcomponents have to know anything about the state of the parent. 
- Ports `UploadChip` to ts :3 

Screen Shot 2023-01-30 at 4 50 29 PM

### Dragons

None

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/web/src/components/upload/UploadChip.tsx [hunk 1]
```diff
@@ -17,7 +16,25 @@ const messages = {
   firstArtistPlaylist: 'Upload Your First Playlist'
 }
 
-const UploadChip = ({ type, variant, isArtist = false, isFirst, onClick }) => {
+type UploadChipProps = {
+  type?: 'track' | 'album' | 'playlist'
+  // nav: For display in a nav-like column
+  // card: Looks like a 'Card'
+  // tile: Looks like a 'TrackTile'
```
Comment (nicoback2) [inline packages/web/src/components/upload/UploadChip.tsx:23]: Nit: might want to use JSDoc style comments to make these comments show up with intellisense

```suggestion
  /**
   * nav: For display in a nav-like column
   * card: Looks like a 'Card'
   * tile: Looks like a 'TrackTile'
   */
```

Comment (nicoback2) [reply packages/web/src/components/upload/UploadChip.tsx:23]: but also ik you were just porting the proptypes so nbd

**Review by nicoback2 — APPROVED** [review]
thanks!!!

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/piazz-fix-upload-chip

**Pull request merged**